docs/getting-started.md

```diff
@@ -5,119 +5,147 @@
+The installation process will take 30-45 minutes to configure and provision all the resources (this will vary based on the options you've selected). Detailed logs from the installation process are stored in **saas-boost-install.log**. 
 
 ## Step 4 - Login to AWS SaaS Boost
-After the installation completes, you will receive a message at the email address you provided during the installation process. This message includes a link with a URL to the AWS SaaS Boost administration application. The message appears as follows:
+As part of the installation process you will receive a message at the email address you provided during the installation process. This message includes a link with a URL to the AWS SaaS Boost administration application. The message appears as follows:
 
 ![Welcome Email](images/gs-welcome-email.png?raw=true "Welcome Email")
 
-Copy the temporary password that is shown here. Select the link to the AWS SaaS Boost administration application. The following login appears:
+Copy the temporary password that is shown here. Once the install script completes, it will also print out the same URL of the AWS SaaS Boost administration application. Use your web browser to navigate to the admin application. The following login appears:
 
 ![Login Screen](images/gs-login.png?raw=true "Login Screen")
 
 Enter `admin` for the user name and enter the temporary password that was delivered in the email referenced earlier. Since you're logging in with a temporary password, you're prompted to enter a new password for your account. The screen appears as follows:
 
 ![Change Password](images/gs-change-password.png?raw=true "Change Password")
 
-## Step 5 - Configure Application Settings
-After you've logged in to the AWS SaaS Boost administration application, the first thing you need to do is configure the environment to align it with the needs of your application. To set up the application, select **Application** from the navigation on the left side of the screen. A page appears similar to the following:
+## Step 5 - Configure Tiers and Application Settings
+After you've logged in to the AWS SaaS Boost administration application, the first thing you need to do is configure the environment to align it with the needs of your application.
+
+AWS SaaS Boost supports configuring your application settings based on _tiers_. Tiers allow you to package your SaaS offering for different segments of customers. For example, you may have a trial tier, or a premium tier in addition to your standard tier offering. AWS SaaS Boost creates a ***default*** tier for you. If you'd like to rename the default tier, or create additional tiers, select **Tiers** from the navigation on the left side of the screen. A page appears similar to the following:
+
+![Application Setup](images/gs-tiers.png?raw=true "Tiers")
+
+Click on the **Create Tier** button to make new tiers or click on the tier name in the table listing to edit existing tiers.
```

Comment:
We should have something here that describes what marking a tier as default means. 

e.g. Tiers can be marked as "default", which means that unset application configurations for non-default tiers will be automatically filled in by the default tier's configuration.